hibike/travis/Makefile

```diff
@@ -12,7 +12,7 @@ artifacts-install:
 	$(nop)
 
 lint:
-	cd .. && pylint *.py
+	cd .. && find | grep "\\.py$$" | xargs pylint
```

Comment:
Ah, I didn't realize that `**` only globs recursively by default on zsh; on bash, you'd need to set `globstar`. Does `find . -name "*.py" | xargs pylint` work?